app/scripts/components/common/map/style-generators/hooks.ts

```diff
@@ -38,11 +43,19 @@ export function useZarr({ id, stacCol, stacApiEndpointToUse, date, onStatusChang
           controller
         });
 
-        setAssetUrl(data.assets.zarr.href);
+        const tileParams = {
```

Comment:
🔧 : This matches the local state variable's name, can we change this so it isn't confusing? Or i'd just set state directly with the object like so...
```
        if (data.assets.zarr.href) {
          setTileParams({
             url: data.assets.zarr.href,
             time_slice: date,
             ...sourceParams
           });
        }
```